docs/key-concepts.md

```diff
@@ -1,6 +1,6 @@
 # Key concepts
 
-The Green Web Foundation offers an API to provide any IP address or domain, and get back a 'green' result indicating that the site or digital service appears to be running on digital powered by green energy, and we have found evidence to support this claim or a 'grey' result where no evidence was found for green energy.
```

Comment:
I think it's ok to have the 'The' here to be honest, as we're talking about the foundation offering an API, and it's sentence in a paragraph, rather than a the name of the organistion all by itself. Personally, I think it reads a bit funny when we remove the article. This is consistence with the usage I've outlined in the article, where the Mozilla Foundation, another foundation is totally with beginning sentences with 'The' if it makes them read better. I've linked directly to the example below:

https://docs.google.com/document/d/1ag3WJ01saMP0NO1-WHD0xq0DpC9koTzVnH3I8OUUWU8/edit?tab=t.0#heading=h.t4lcbe6foxja